conf/vmsync.env

```diff
@@ -0,0 +1,6 @@
+POSTGRES_USER=vmaas_gatherer
+POSTGRES_PASSWORD=vmaas_gatherer_pwd
+LOGGING_LEVEL=INFO
+API_RETRIES=3
+PAGE_SIZE=5000
+BASE_URL=http://localhost:8080/api/v3/cves
```

Comment:
I'd just set `https://console.redhat.com/api/vmaas/v3/cves` here, not relying on local vmaas

Reply:
also newline pls